lib/eslint-tester.js

```diff
@@ -181,7 +188,16 @@ ESLintTester.prototype = {
         });
 
         return result.suite;
+    },
+
+    /**
+     * Gets the name of the last used parser.
+     * @returns {string} The name of the last used parser.
+     */
+    getParser: function() {
```

Comment:
Just realized this doesn't make sense because each test can have a different parser. :-/ Need to think about this some more.